app/model/view/UserViewFactory.php

```diff
@@ -55,13 +55,17 @@ private function getExternalIds(User $user, bool $canViewAllExternalIds = false)
     return $user->getConsolidatedExternalLogins($filter);
   }
 
+  private function isUserLoggedInUser(User $user) {
```

Comment:
This seems like something which should be checked by our authorization implementation, is it possible there?